configuration/page.md

```diff
@@ -49,12 +49,10 @@ See [folder configuration](folder.md) for details on how to configure a folder.
 
 ---
 
-## author
+## author / authors
```

Comment:
How about we remove `/ authors` to keep it consistent with `tags`, `category` ?
Alternatively, we can revise them to include singular/plural form too.